osu.Game/Screens/Edit/Compose/Components/Timeline/Timeline.cs

```diff
@@ -196,7 +196,7 @@ protected override void Update()
             {
                 defaultTimelineZoom = getZoomLevelForVisibleMilliseconds(6000);
 
-                float initialZoom = (float)(defaultTimelineZoom * editorBeatmap.BeatmapInfo.TimelineZoom);
+                float initialZoom = (float)(defaultTimelineZoom * (editorBeatmap.BeatmapInfo.TimelineZoom <= 0 ? 1 : editorBeatmap.BeatmapInfo.TimelineZoom));
```

Comment:
There's no negative timeline zoom, that's guaranteed by `LegacyBeatmapDecoder`.
```suggestion
                float initialZoom = (float)(defaultTimelineZoom * (editorBeatmap.BeatmapInfo.TimelineZoom == 0 ? 1 : editorBeatmap.BeatmapInfo.TimelineZoom));
```